### PR TITLE
Add uber support

### LINKED ISSUE #144: Uber API integration request
In october, Pizzly teams up with Hacktoberfest 2020 to increase the number of API integrations to up to 100 APIs 🎉 To learn more about how to participate and win, look at [our Hacktoberfest guide](https://github.com/Bearer/Pizzly/blob/master/HACKTOBERFEST.md).

**Uber** has been identified as a major API that we would love to support. Your mission is to create a configuration file that make this API ready to be used by Pizzly developers.

Here are a few links to help you achieve this task:

* [Uber official website](https://www.uber.com/)
* [Uber Developers Portal](https://developer.uber.com/)
* [Uber's OAuth documentation](https://developer.uber.com/docs/riders/guides/authentication/user-access-token)
* [Pizzly's guide on how to add a new integration](https://github.com/Bearer/Pizzly/wiki/Reference-:-Integrations#adding-a-new-integration)

If you are interested in adding this integration to Pizzly, please comment below first, to tell other contributors that you're on it. 

Once you've the configuration ready, please create a PR that links to this issue ([more detail in our Hacktoberfest guide](https://github.com/Bearer/Pizzly/blob/master/HACKTOBERFEST.md)). Here's an [example of a similar PR](https://github.com/Bearer/Pizzly/pull/99/files) that was recently accepted.

Happy hacking! 👨‍🎤

### PR DESCRIPTION
**Pull request by ry-v1**
## Proposed changes

Add uber api support. Fixes #144

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x ] I have read the [CONTRIBUTING](https://github.com/bearer/pizzly/blob/master/README.md#contributing-guide) guide
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: integrations/uber.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "authType": "OAUTH2",
+    "tokenParams": {},
+    "authorizationParams": { "response_type": "code", "grant_type": "authorization_code" }
+  },
+  "request": {
+    "baseURL": "https://api.uber.com/v1.2",
+    "headers": {
+      "Accept": "application/json",
+      "Authorization": "Bearer ${auth.accessToken}",
+      "User-Agent": "Pizzly (https://github.com/bearer/pizzly, 0.2.0)"
```
Comment (Frenchcooc) [inline integrations/uber.json:15]:
```suggestion
      "User-Agent": "Pizzly"
```

While I presume you had something great in mind to add both a link to the repo and the version number, sounds better to keep this header simple 🙏  Others configurations are using the proposed change.

**Review by Frenchcooc — CHANGES_REQUESTED** [review]

**Review by Frenchcooc — CHANGES_REQUESTED** [review]
User-agent needs to be reviewed IMHO

File: integrations/uber.json (hunk 1 above)
Comment (ry-v1) [reply integrations/uber.json]: In the issue #144 description, there is an example PR(Discord API) which has link to the repo and the version number. So, I assumed that is standard. Anyways made the suggested change. Thanks

**Review by Frenchcooc — APPROVED** [review]
My bad! Will fix the discord configuration file

**Pull request merged**